kore/src/Kore/Options.hs

```diff
@@ -77,10 +77,12 @@ data KoreParserOptions = KoreParserOptions
 
 parsePatternOptions :: Parser PatternOptions
 parsePatternOptions = PatternOptions
-    <$> strOption
-        (  metavar "PATTERN_FILE"
-        <> long "pattern"
-        <> help "Kore pattern source file to parse [and verify]. Needs --module.")
+    <$> some (
+            strOption
+            (  metavar "PATTERN_FILE"
+            <> long "pattern"
+            <> help "Kore pattern source file to parse [and verify]. Needs --module.")
+        )
```

Comment:
Let's extract this as `parsePatternFileName` so that the option parser simply reads `many parsePatternFileName`.